src/analysis/scripts/export_connectivity.sh

```diff
@@ -149,10 +153,13 @@ then
         # NOTE: disabled for now, because large and not that useful
         # ec_export_table_shp "${OUTPUT_DIR}" "neighborhood_connected_census_blocks"
 
+        # Export neighborhood_score_inputs as CSV
+        ec_export_table_csv "${OUTPUT_DIR}" "neighborhood_score_inputs"
+
         # Export neighborhood_overall_scores as CSV
         ec_export_table_csv "${OUTPUT_DIR}" "neighborhood_overall_scores"
         # Send overall_scores to Django app
-        update_overall_scores "${OUTPUT_DIR}/neighborhood_overall_scores.csv"
+        update_overall_scores "${OUTPUT_DIR}/neighborhood_score_inputs.csv"
```

Comment:
In your PR comment you stated that the new overall_scores is the one that should be displayed on the site. Do we need to make this intermediate table available and if so for what purpose?